### PR TITLE
Evercrypt submodule

### PR DESCRIPTION
**Pull request by achamayou**

### CODE REVIEW

**Comment by jumaffre** [comment]
We should update our documentation so that users know to `git clone --recursive`.

File: 3rdparty/evercrypt [hunk 1]
```diff
@@ -0,0 +1 @@
+../evercrypt-msr/evercrypt/
```
Comment (jumaffre) [inline 3rdparty/evercrypt]: Is it bad practice to have a submodule live in `3rdpary/`?

Comment (achamayou) [reply 3rdparty/evercrypt]: No but we can't submodule just the directory, so it would be quite messy.

**Comment by cimetrics** [comment]
images

**Pull request merged**